Resources/Computing/SSHComputingElement.py

```diff
@@ -323,6 +393,10 @@ def _reset(self):
     if not self.workArea.startswith('/'):
       self.workArea = os.path.join(self.sharedArea, self.workArea)
 
+    self.parallelLibrary = self.ceParameters.get('ParallelLibrary')
+    if self.parallelLibrary:
+      self.loadParallelLibrary('.')
```

Comment:
`loadParallelLibrary` returns S_OK/S_ERROR, but here you don't treat what's returned